apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-authorization.tsx

```diff
@@ -94,7 +97,7 @@ export const AddAPIResourceAuthorization: FunctionComponent<AddAPIResourceAuthor
                 content={ 
                     (<>
                         { t("extensions:develop.apiResource.wizard.addApiResource.steps.authorization.form." + 
-                            "rbacMessage") }
+                            "rbacMessage", { productName: productName }) }
```

Comment:
When the key and variable name are same, we can add the variable only inside curly braces and js will automatically covert it to a key value pair.

```suggestion
                            "rbacMessage", { productName }) }
```

Let's update the other places as well :)

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4369/commits/5f9e86491ef0a3af12f7235bb143cadc4347c5b8